backend/src/services/project-membership/project-membership-service.ts

```diff
@@ -72,82 +74,6 @@ export const projectMembershipServiceFactory = ({
     return projectMembershipDAL.findAllProjectMembers(projectId);
   };
 
-  const inviteUserToProject = async ({ actorId, actor, actorOrgId, projectId, emails }: TInviteUserToProjectDTO) => {
```

Comment:
How come this was deleted?

Reply:
This was not used anymore after daniel migration to ghost user